loki-config.yaml

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
I prefer not to pollute the repository with custom configuration files like this, could you create a separate docs/ entry (see the structure of the Nomad and Kubernetes docs) describing this?

Reply:
Probably rebase first :)